Resources/Prototypes/Recipes/Construction/structures.yml

```
@@ -1456,6 +1456,25 @@
   # Same here. - 20kdc
     - !type:TileNotBlocked
 
+- type: construction
+  name: emergency light
+  id: EmergencyLightFixture
+  graph: LightFixture
+  startNode: start
+  targetNode: emergencyLight
+  category: construction-category-structures
+  description: An emergency light.
+  icon:
+    sprite: Structures/Wallmounts/Lighting/emergency_light.rsi
+    state: base
+  objectType: Structure
+  placementMode: SnapgridCenter
+  canRotate: true
+  canBuildInImpassable: false
+  conditions:
+  # Same here. - 20kdc
+    - !type:TileNotBlocked
```

Comment is an unnecessary copypaste
```suggestion
  conditions:
    - !type:TileNotBlocked
```